wurst/_items/HealingItems.wurst

```diff
@@ -0,0 +1,51 @@
+package HealingItems
+
+import ID
+import HashMap
+import ClosureEvents
+import HealingSystem
+
+HashMap<int, real> itemSelfHealAmount = null
+HashMap<int, real> itemTargetedHealAmount = null
+
+let ABILITY_HEALPOTION_1 = 'A0DO'
```

Comment:
Don't use these. Just use `ABILITY_HEAL_POTION` and `65`. The others are deprecated.